index.html

```diff
@@ -198,7 +198,11 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
         return height / 3;
     };
     </script>
+
```

Comment:
What was the point of these blank lines added here? :) Watch your commits to not get them littered by unnecessary code. Use diff command for this purpose.